server/service/transport_error.go

```diff
@@ -62,10 +62,12 @@ type existsErrorInterface interface {
 
 // encode error and status header to the client
 func encodeError(ctx context.Context, err error, w http.ResponseWriter) {
+	ctxerr.Handle(ctx, err)
```

Comment:
Now that we are using it, does it make sense for `Handle` to return an `error`?

Reply:
Yeah, not really, I kept it from the old experiment but I don't think it will be needed, I'll remove it.